pkg/dataplane/util/ansibleee.go

```diff
@@ -45,9 +45,6 @@ type EEJob struct {
 	Args []string `json:"args,omitempty"`
 	// NetworkAttachments is a list of NetworkAttachment resource names to expose the services to the given network
 	NetworkAttachments []string `json:"networkAttachments,omitempty"`
-	// PreserveJobs - do not delete jobs after they finished e.g. to check logs
-	// PreserveJobs default: true
-	PreserveJobs bool `json:"preserveJobs,omitempty"`
```

Comment:
need to drop it from openstackdataplanedeployment_types.go and pkg/dataplane/util/ansible_execution.go also

Reply:
Right, I had to rebase on main. I wasn't seeing any of these in my branch

```
❯ rg -i preservejobs -l
docs/assemblies/ctlplane_resources.adoc
apis/bases/core.openstack.org_openstackcontrolplanes.yaml
config/crd/bases/core.openstack.org_openstackcontrolplanes.yaml
```

Rebased, removing and re-testing now. New revision of this incoming.